File: PYTHON/PRAK404-2210817220029-SitiAinurRahmawati.py
while True :
    print("Pilih Program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    pilihan=int(input("Masukkan Pilihan : "))
    if pilihan==5 :
        print("Terimakasih telah menggunakan kalkulator SITI AINUR RAHMAWATI")
        break
    elif pilihan>= 1 and pilihan <= 4 :
        pertama=int(input("Masukkan nilai pertama : "))
        kedua=int(input("Masukkan nilai kedua : "))
        if pilihan==1 :
            hasil= pertama + kedua
            print("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f" %(pertama, kedua, hasil))
        elif pilihan==2 :
            hasil= pertama - kedua
            print("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f" %(pertama, kedua, hasil))
        elif pilihan==3 :
            hasil= pertama * kedua
            print("Hasil perkalian antara %.2f dengan %.2f adalah %.2f" %(pertama, kedua, hasil))
        elif pilihan==4 :
            hasil= pertama / kedua
            print("Hasil pembagian antara %.2f dengan %.2f adalah %.2f" %(pertama, kedua, hasil))
    else :
        print("Input anda salah, silahkan coba lagi")

File: C/PRAK404-2210817220029-SitiAinurRAhmawati.c
#include <stdio.h>
int main () {
    float pilihan, pertama, kedua, hasil;
    while (1) {
    printf("Pilih Program");
    printf("\n1. Penjumlahan");
    printf("\n2. Pengurangan");
    printf("\n3. Perkalian");
    printf("\n4. Pembagian");
    printf("\n5. Exit");
    printf("\nMasukkan Pilihan : ");
    scanf("%f", &pilihan);
    if (pilihan==5) {
        printf("Terimakasih telah menggunakan kalkulator SITI AINUR RAHMAWATI");
        break;}
    else if (pilihan>= 1 && pilihan <= 4) {
        printf("Masukkan nilai pertama : ");
        scanf("%f", &pertama);
        printf("Masukkan nilai kedua : ");
        scanf("%f", &kedua);
        if (pilihan==1) {
            hasil= pertama + kedua;
            printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n", pertama, kedua, hasil);
        } else if (pilihan==2) {
            hasil= pertama - kedua;
            printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n", pertama, kedua, hasil);
        } else if (pilihan==3) {
            hasil= pertama * kedua;
            printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n", pertama, kedua, hasil);
        } else if (pilihan==4) {
            hasil= pertama / kedua;
            printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n", pertama, kedua, hasil);
        }
    } else {
        printf("Input anda salah, silahkan coba lagi\n");
    }
    }
}